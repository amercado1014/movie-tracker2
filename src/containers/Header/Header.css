.header {
  position: fixed;
  top: 0;
  height: 100px;
  z-index: 97;
  background-color: #001021;
  /* display: flex; */
  /* justify-content: flex-start */
}

.header h1 {
  color: #f5f5f5;
}

.welcome {
  position: absolute;
  top: -10px;
  right: 10px;
}

h5 {
  color: #f5f5f5;
}

.nav {
  display: flex;
  justify-content: center;
  background-color: #034748;
  /* position: absolute; */
  width: 100vw;
  /* right: 10px; */
  z-index: 98;;
}

.nav-link-wrapper {
  display: inline-block;
  height: 18px;
  /* width: 100px; */
  padding: 10px 20px;
  /* margin: 25px; */
  /* border: 1px solid black; */
  background-color: #034748;
}

a {
  color: #f5f5f5;
  background-color:#034748;
  padding: 5px;
  text-decoration: none;
}

.active {
  color: #11B5E4;
  /* background-color: #1481BA; */
}
