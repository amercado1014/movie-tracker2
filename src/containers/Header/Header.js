import React from 'react';
import { NavLink, withRouter } from 'react-router-dom';
import { connect } from 'react-redux';
import { logOut } from '../../actions';


const Header = (props) => {
  
  const handleClick = (event) => {
    console.log(props)
    event.preventDefault();
    props.logOut();
  };

  return (
    <div>
      <NavLink to='/'><h1>Movie Tracker</h1></NavLink>
      <NavLink to='/login'>Login</NavLink>
      <button onClick={handleClick}>Logout</button>
      <NavLink to='/signup'>Sign Up</NavLink>
    </div>
  );
};

const mapStateToProps = state => ({
  user: state.user
});

const mapDispatchToProps = dispatch => ({
  logOut: () => dispatch(logOut())
});

export default withRouter(connect(mapStateToProps, mapDispatchToProps)(Header));  