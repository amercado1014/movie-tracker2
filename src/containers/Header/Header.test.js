import React from 'react';
import ReactDOM from 'react-dom';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header when logged in', () => {

  let wrapper;
  let mockUser;

  it('matches the snapshot', () => {
    const mockUser = { name: "me" };
    const wrapper = shallow(<Header user={mockUser} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should call logOut and resetFavorites on handleClick', () => {
    mockUser = { name: "me" };
    const mockLogOut = jest.fn();
    const mockResetFavorites = jest.fn();
    wrapper = shallow(<Header user={mockUser} 
      logOut={mockLogOut} 
      resetFavorites={mockResetFavorites} />);
    wrapper.instance().handleClick();
    expect(mockLogOut).toHaveBeenCalled();
    expect(mockResetFavorites).toHaveBeenCalled();
  });

});

describe('Header when logged out', () => {

  it('matches the snapshot', () => {
    const mockUser = {};
    const wrapper = shallow(<Header user={mockUser} />);
    expect(wrapper).toMatchSnapshot();
  });

});