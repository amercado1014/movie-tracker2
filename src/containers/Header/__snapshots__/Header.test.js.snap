// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header when logged in matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    user={
        Object {
            "name": "me",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Movie Tracker
</h1>,
        <div
          className="nav"
>
          <div
                    className="welcome"
          >
                    <h3>
                              Welcome 
                              me
                    </h3>
          </div>
          <div
                    className="nav-link-wrapper"
          >
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              exact={true}
                              to="/"
                    >
                              Movies
                    </NavLink>
          </div>
          <div
                    className="nav-link-wrapper"
          >
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              onClick={[Function]}
                              to="/login"
                    >
                              Logout
                    </NavLink>
          </div>
          <div
                    className="nav-link-wrapper"
          >
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              to="/favorites"
                    >
                              Favorites
                    </NavLink>
          </div>
</div>,
        <h2 />,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Movie Tracker",
        },
        "ref": null,
        "rendered": "Movie Tracker",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="welcome"
>
              <h3>
                            Welcome 
                            me
              </h3>
</div>,
            <div
              className="nav-link-wrapper"
>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            exact={true}
                            to="/"
              >
                            Movies
              </NavLink>
</div>,
            <div
              className="nav-link-wrapper"
>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            onClick={[Function]}
                            to="/login"
              >
                            Logout
              </NavLink>
</div>,
            <div
              className="nav-link-wrapper"
>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            to="/favorites"
              >
                            Favorites
              </NavLink>
</div>,
          ],
          "className": "nav",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h3>
                Welcome 
                me
</h3>,
              "className": "welcome",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "Welcome ",
                  "me",
                ],
              },
              "ref": null,
              "rendered": Array [
                "Welcome ",
                "me",
              ],
              "type": "h3",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                ariaCurrent="true"
                exact={true}
                to="/"
>
                Movies
</NavLink>,
              "className": "nav-link-wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Movies",
                "exact": true,
                "to": "/",
              },
              "ref": null,
              "rendered": "Movies",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                ariaCurrent="true"
                onClick={[Function]}
                to="/login"
>
                Logout
</NavLink>,
              "className": "nav-link-wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Logout",
                "onClick": [Function],
                "to": "/login",
              },
              "ref": null,
              "rendered": "Logout",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/favorites"
>
                Favorites
</NavLink>,
              "className": "nav-link-wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Favorites",
                "to": "/favorites",
              },
              "ref": null,
              "rendered": "Favorites",
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Movie Tracker
</h1>,
          <div
            className="nav"
>
            <div
                        className="welcome"
            >
                        <h3>
                                    Welcome 
                                    me
                        </h3>
            </div>
            <div
                        className="nav-link-wrapper"
            >
                        <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    exact={true}
                                    to="/"
                        >
                                    Movies
                        </NavLink>
            </div>
            <div
                        className="nav-link-wrapper"
            >
                        <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    onClick={[Function]}
                                    to="/login"
                        >
                                    Logout
                        </NavLink>
            </div>
            <div
                        className="nav-link-wrapper"
            >
                        <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    to="/favorites"
                        >
                                    Favorites
                        </NavLink>
            </div>
</div>,
          <h2 />,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Movie Tracker",
          },
          "ref": null,
          "rendered": "Movie Tracker",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="welcome"
>
                <h3>
                                Welcome 
                                me
                </h3>
</div>,
              <div
                className="nav-link-wrapper"
>
                <NavLink
                                activeClassName="active"
                                ariaCurrent="true"
                                exact={true}
                                to="/"
                >
                                Movies
                </NavLink>
</div>,
              <div
                className="nav-link-wrapper"
>
                <NavLink
                                activeClassName="active"
                                ariaCurrent="true"
                                onClick={[Function]}
                                to="/login"
                >
                                Logout
                </NavLink>
</div>,
              <div
                className="nav-link-wrapper"
>
                <NavLink
                                activeClassName="active"
                                ariaCurrent="true"
                                to="/favorites"
                >
                                Favorites
                </NavLink>
</div>,
            ],
            "className": "nav",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3>
                  Welcome 
                  me
</h3>,
                "className": "welcome",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "Welcome ",
                    "me",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  "Welcome ",
                  "me",
                ],
                "type": "h3",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  exact={true}
                  to="/"
>
                  Movies
</NavLink>,
                "className": "nav-link-wrapper",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": "Movies",
                  "exact": true,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Movies",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  onClick={[Function]}
                  to="/login"
>
                  Logout
</NavLink>,
                "className": "nav-link-wrapper",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": "Logout",
                  "onClick": [Function],
                  "to": "/login",
                },
                "ref": null,
                "rendered": "Logout",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  to="/favorites"
>
                  Favorites
</NavLink>,
                "className": "nav-link-wrapper",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": "Favorites",
                  "to": "/favorites",
                },
                "ref": null,
                "rendered": "Favorites",
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`Header when logged out matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    user={Object {}}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Movie Tracker
</h1>,
        <div
          className="nav"
>
          <div
                    className="welcome"
          >
                    <h3>
                              You are logged out.
                    </h3>
          </div>
          <div
                    className="nav-link-wrapper"
          >
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              exact={true}
                              to="/"
                    >
                              Movies
                    </NavLink>
          </div>
          <div
                    className="nav-link-wrapper"
          >
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              to="/login"
                    >
                              Login
                    </NavLink>
          </div>
          <div
                    className="nav-link-wrapper"
          >
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              to="/signup"
                    >
                              Sign Up
                    </NavLink>
          </div>
</div>,
        <h2 />,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Movie Tracker",
        },
        "ref": null,
        "rendered": "Movie Tracker",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="welcome"
>
              <h3>
                            You are logged out.
              </h3>
</div>,
            <div
              className="nav-link-wrapper"
>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            exact={true}
                            to="/"
              >
                            Movies
              </NavLink>
</div>,
            <div
              className="nav-link-wrapper"
>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            to="/login"
              >
                            Login
              </NavLink>
</div>,
            <div
              className="nav-link-wrapper"
>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            to="/signup"
              >
                            Sign Up
              </NavLink>
</div>,
          ],
          "className": "nav",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h3>
                You are logged out.
</h3>,
              "className": "welcome",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "You are logged out.",
              },
              "ref": null,
              "rendered": "You are logged out.",
              "type": "h3",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                ariaCurrent="true"
                exact={true}
                to="/"
>
                Movies
</NavLink>,
              "className": "nav-link-wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Movies",
                "exact": true,
                "to": "/",
              },
              "ref": null,
              "rendered": "Movies",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/login"
>
                Login
</NavLink>,
              "className": "nav-link-wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Login",
                "to": "/login",
              },
              "ref": null,
              "rendered": "Login",
              "type": [Function],
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/signup"
>
                Sign Up
</NavLink>,
              "className": "nav-link-wrapper",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": "Sign Up",
                "to": "/signup",
              },
              "ref": null,
              "rendered": "Sign Up",
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Movie Tracker
</h1>,
          <div
            className="nav"
>
            <div
                        className="welcome"
            >
                        <h3>
                                    You are logged out.
                        </h3>
            </div>
            <div
                        className="nav-link-wrapper"
            >
                        <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    exact={true}
                                    to="/"
                        >
                                    Movies
                        </NavLink>
            </div>
            <div
                        className="nav-link-wrapper"
            >
                        <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    to="/login"
                        >
                                    Login
                        </NavLink>
            </div>
            <div
                        className="nav-link-wrapper"
            >
                        <NavLink
                                    activeClassName="active"
                                    ariaCurrent="true"
                                    to="/signup"
                        >
                                    Sign Up
                        </NavLink>
            </div>
</div>,
          <h2 />,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Movie Tracker",
          },
          "ref": null,
          "rendered": "Movie Tracker",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="welcome"
>
                <h3>
                                You are logged out.
                </h3>
</div>,
              <div
                className="nav-link-wrapper"
>
                <NavLink
                                activeClassName="active"
                                ariaCurrent="true"
                                exact={true}
                                to="/"
                >
                                Movies
                </NavLink>
</div>,
              <div
                className="nav-link-wrapper"
>
                <NavLink
                                activeClassName="active"
                                ariaCurrent="true"
                                to="/login"
                >
                                Login
                </NavLink>
</div>,
              <div
                className="nav-link-wrapper"
>
                <NavLink
                                activeClassName="active"
                                ariaCurrent="true"
                                to="/signup"
                >
                                Sign Up
                </NavLink>
</div>,
            ],
            "className": "nav",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <h3>
                  You are logged out.
</h3>,
                "className": "welcome",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "You are logged out.",
                },
                "ref": null,
                "rendered": "You are logged out.",
                "type": "h3",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  exact={true}
                  to="/"
>
                  Movies
</NavLink>,
                "className": "nav-link-wrapper",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": "Movies",
                  "exact": true,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Movies",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  to="/login"
>
                  Login
</NavLink>,
                "className": "nav-link-wrapper",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": "Login",
                  "to": "/login",
                },
                "ref": null,
                "rendered": "Login",
                "type": [Function],
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  to="/signup"
>
                  Sign Up
</NavLink>,
                "className": "nav-link-wrapper",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": "Sign Up",
                  "to": "/signup",
                },
                "ref": null,
                "rendered": "Sign Up",
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
